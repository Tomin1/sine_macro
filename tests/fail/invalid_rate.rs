/*
 * Copyright (c) 2025 Tomi Lepp√§nen
 * SPDX-License-Identifier: MIT
 */

use sine_macro::sine_wave;

fn main() {
    let _wave = sine_wave!(rate: 100);
}
