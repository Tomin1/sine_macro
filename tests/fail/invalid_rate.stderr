error: `rate` should be more than 1760 for frequency of 440 Hz
 --> tests/fail/invalid_rate.rs:9:34
  |
9 |     let _wave = sine_wave!(rate: 1000);
  |                                  ^^^^
